imap_processing/cdf/config/imap_glows_global_cdf_attrs.yaml

```
@@ -0,0 +1,27 @@
+Descriptor: &desc
+  GLOWS>GLObal Solar Wind Structure
+TEXT: &txt >
+  The GLObal Solar Wind Structure (GLOWS) is a non-imaging single-pixel Lyman-alpha
+  photometer that will be used to observe the sky distribution of the helioglow to
+  better understand the evolution of the solar wind structure.
+  The Lyman-alpha photon counts from these observations can be used to build a more
+  comprehensive picture of the solar wind structure and how it changes through the
+  solar cycles.
+  GLOWS design and assembly is led by the Space Research Center, Warsaw, Poland
+  (CBK PAN). See https://imap.princeton.edu/instruments/glows for more details.
```

Don't forget to add this here and other areas. This has been causing issues with generating valid CDF files.
```suggestion
  (CBK PAN). See https://imap.princeton.edu/instruments/glows for more details.
Instrument_type: Particles (space)
```